### PR TITLE
Add wp-bones extension

### PR DESCRIPTION
**Pull request by gfazioli**
## Description

This mod provides several commands centered around managing the WP Bones WordPress plugin framework. It provides quite a few commands for varying uses, so there should be something for everyone! Take a look at the scenarios below to get a feel for how you might use the extension.

## Features

- **Search in the WP Bones Documentation**: Search the WP Bones documentation for a specific term.
- **Create a New Plugin**: Create a new plugin using the WP Bones template.
- **Menu bar commands**: Quickly access the most common commands from the menu bar. And check the latest version of WP Bone

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/wp-bones/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "plugin",
+    "template",
+    "documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "menu-bar",
+      "title": "WP Bones",
+      "subtitle": "WP Bones",
```
Comment (pernielsentikaer) [inline extensions/wp-bones/package.json:25]:
```suggestion
      "title": "WP Bones Menu Bar",
      "subtitle": "WP Bones",
```

**Comment by pernielsentikaer** [comment]
Should make some padding inside the Grid.Item and maybe use an icon set to make it look even more nice?

**Comment by gfazioli** [comment]
> Should make some padding inside the Grid.Item and maybe use an icon set to make it look even more nice?

I'll do soon, thx 🙏

**Comment by gfazioli** [comment]
@pernielsentikaer icons fixed and compressed 
image
image

**Pull request merged**